Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Extensions/FbDbContextOptionsExtensions.cs

```diff
@@ -28,15 +28,15 @@ public static class FbDbContextOptionsExtensions
 	{
 		public static DbContextOptionsBuilder UseFirebird(this DbContextOptionsBuilder optionsBuilder, string connectionString, Action<FbDbContextOptionsBuilder> fbOptionsAction = null)
 		{
-			var extension = GetOrCreateExtension(optionsBuilder).WithConnectionString(connectionString);
+			var extension = (FbOptionsExtension)GetOrCreateExtension(optionsBuilder).WithConnectionString(connectionString);
```

Comment:
What's the reason for the casting?

Reply:
You will not be able to recover the connection properties for example!

With this -> options.FindExtension<FbOptionsExtension>() in FbOptions


And I already tested !!!

Reply:
Ahh, now I get it. The email notification missed the images. The `WithConnection` is fucking the type.

Reply:
As I explained, we can use this CAST!